doc/Language/variables.pod6

```diff
@@ -498,6 +399,212 @@ augment slang Regex {  # derive from $~Regex and then modify $~Regex
 }
 =end code
 
+
+=head1 Item assignment, list assignment, and destructuring assignment
+
+There are three types of variable assignment, I<item assignment>, I<list
```

Comment:
Not totally clear why this is not side by side with the other. That implies it's going to take longer to see what was taken out, and what was left. In general, we're not very fond of totally eliminating old documentation, for basically the same reason we're not fond of eliminating old code unless there's a very good reason for doing it.